src/agent/README.md

```diff
@@ -43,12 +43,16 @@ The `rust-agent` depends on [`grpc-rs`](https://github.com/pingcap/grpc-rs) by P
 - https://github.com/alipay/grpc-rs/tree/rust_agent
 
 ### Build from Source
-The rust-agent need to be built with rust nightly, and static linked with musl.
+The rust-agent need to be built with static linked with musl:
 ```bash
-rustup target add x86_64-unknown-linux-musl
-git submodule update --init --recursive  
-sudo ln -s /usr/bin/g++ /bin/musl-g++  
-cargo build --target x86_64-unknown-linux-musl --release
+$ rustup target add x86_64-unknown-linux-musl
```

Comment:
Kata works on multiple architectures, so how about:

```suggestion
$ arch=$(uname -m)
$ [ "$arch" == "ppc64le" ] && arch=powerpc64le
$ rustup target add "${arch}-unknown-linux-musl"
```
Note the ppc64 special-casing - see:

- https://github.com/kata-containers/tests/blob/2.0-dev/.ci/install_rust.sh
- https://github.com/kata-containers/tests/blob/master/.ci/kata-arch.sh

Reply:
Thank you for suggestions